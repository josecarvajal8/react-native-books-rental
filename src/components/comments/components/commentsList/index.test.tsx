import React from 'react';
import renderer from 'react-test-renderer';
import CommentsList from '.';
const mockProps = {
    data: []
}
test('Render commentsList component correctly', () => {
    const commentsListCmp = renderer.create(<CommentsList  {...mockProps} />).toJSON();
    expect(commentsListCmp).toMatchSnapshot();
});