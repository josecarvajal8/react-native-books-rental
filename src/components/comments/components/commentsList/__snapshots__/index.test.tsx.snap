// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render commentsList component correctly 1`] = `
<RCTScrollView
  bounces={false}
  data={Array []}
  disableVirtualization={false}
  getItem={[Function]}
  getItemCount={[Function]}
  horizontal={false}
  initialNumToRender={10}
  keyExtractor={[Function]}
  maxToRenderPerBatch={10}
  onContentSizeChange={[Function]}
  onEndReachedThreshold={2}
  onLayout={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  removeClippedSubviews={false}
  renderItem={[Function]}
  scrollEnabled={true}
  scrollEventThrottle={50}
  showsVerticalScrollIndicator={false}
  stickyHeaderIndices={Array []}
  updateCellsBatchingPeriod={50}
  viewabilityConfigCallbackPairs={Array []}
  windowSize={21}
>
  <View />
</RCTScrollView>
`;
